app/Length.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace ConverterApp;
+
+class Length extends Unit
+{
+  public function __construct($longName,$shortName)
+  {
+  	
+    $this->longName = $longName;
```

Comment:
If these two are `Unit` class data member you should simply call its constructor like you did in `Length` class.